### ISSUE TITLE
Separate message type for when the destinationAccount needs to be sent

### ISSUE THREAD

**Issue by emschwartz** [emschwartz/ilp-protocol-paystream#23]
Currently, the data that goes across the wire for every chunk includes a [spot for the `destinationAccount`](https://github.com/emschwartz/ilp-protocol-paystream/blob/master/src/index.ts#L828). At the moment this is only used on the first message the client sends to the server, and is left as an empty string (encoded as a single 0-byte) for every subsequent message.

We should probably make a separate message type that has the `destinationAccount` field.

One question related to this is whether that message should be thought of as a "connect" message, or whether you could send that later, for example if you want to change the ILP address the other party should be sending to.

**Comment by sharafian** [comment]
we definitely want to retain the ability to change the account on an already established stream, so we shouldn't just put it as the connect message.

**Comment by emschwartz** [comment]
Agreed. Then the question is just whether we should have a different type that indicates whether there is an account in there or whether we should keep using the (somewhat hacky) trick of just putting the account as an empty string.
